runtime/kusama/src/governance/old.rs

```diff
@@ -0,0 +1,172 @@
+// Copyright 2022 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
```

Comment:
should sanity check this file and configuration one last time before we merge, to make sure that master config doesn't change.